platform/default/src/mbgl/storage/offline_download.cpp

```diff
@@ -353,12 +353,15 @@ void OfflineDownload::activateDownload() {
 */
 void OfflineDownload::continueDownload() {
     if (resourcesRemaining.empty() && status.complete()) {
+        markPendingUsedResources();
         setState(OfflineRegionDownloadState::Inactive);
         return;
     }
 
+    if (resourcesToBeMarkedAsUsed.size() >= 200) markPendingUsedResources();
```

Comment:
nit: move '200' to constants? Or to some visible place in this file. Maybe same for '64' for batched writes.